unsafe/src/test/java/org/apache/spark/unsafe/bitset/BitSetSuite.java

```diff
@@ -39,6 +39,7 @@ public void basicOps() {
     for (int i = 0; i < bs.capacity(); i++) {
       Assert.assertFalse(bs.isSet(i));
     }
+    Assert.assertFalse(bs.anySet());
```

Comment:
I still think that we should write a proper unit test for `anySet` that will create a new bitset spanning multiple words, then set a single in each word, then check that `anySet()` returns true.  This test isn't sufficient because it wouldn't have caught any of the bugs in earlier iterations of this PR.